File: protocols/line_notify/line_notify_python/simple.py
import requests


url = 'https://notify-api.line.me/api/notify'
token = 'HSjKlkm6ylCcZoHgtaAbk37EZqOdW52lrXuIw1CFqNb'
token_cj = '7LDFKvhJLfzlAbDry9125QaDYTCZCGHLMcA5H5yJMcE'
headers = {'content-type':'application/x-www-form-urlencoded','Authorization':'Bearer '+ token}

message = 'Hello from Python task scheduler.(executed every 10 minutes)'
sticker_package = 11537
sticker_id = 52002735

r = requests.post(url, headers=headers, data = {
    'message':message,
    'stickerPackageId':sticker_package,
    'stickerId':sticker_id,
    })

print (r.text)

File: protocols/line_notify/line_notify_go/simple.go
package main

import (
    "bytes"
    "fmt"
    "net/http"
    "net/url"
)

func main() {
    notifyURL := "https://notify-api.line.me/api/notify"
    token := "HSjKlkm6ylCcZoHgtaAbk37EZqOdW52lrXuIw1CFqNb"
    message := "Hello, World!"
    imageURL := "" // optional image URL

    data := url.Values{}
    data.Set("message", message)
    if imageURL != "" {
        data.Add("imageThumbnail", imageURL)
        data.Add("imageFullsize", imageURL)
    }

    client := &http.Client{}
    req, _ := http.NewRequest("POST", notifyURL, bytes.NewBufferString(data.Encode()))
    req.Header.Add("Content-Type", "application/x-www-form-urlencoded")
    req.Header.Add("Authorization", "Bearer "+token)

    resp, err := client.Do(req)
    if err != nil {
        fmt.Println(err)
        return
    }
    defer resp.Body.Close()

    if resp.StatusCode != 200 {
        fmt.Println("Failed to send notification")
    } else {
        fmt.Println("Notification sent successfully")
    }
}
